<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>entry</class>
 <widget class="QWidget" name="entry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/ico.ico</normaloff>:/res/ico.ico</iconset>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>301</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="name">
      <property name="font">
       <font>
        <family>方正瘦金书简体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="name1"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="code1">
      <property name="font">
       <font>
        <family>方正瘦金书简体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请输入密码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="code11"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="code21"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="code2">
      <property name="font">
       <font>
        <family>方正瘦金书简体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>再次输入密码:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sure">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
